modules/apps/transfer/keeper/grpc_query_test.go

```diff
@@ -247,7 +248,7 @@ func (suite *KeeperTestSuite) TestEscrowAddress() {
 			func() {
 				req = &types.QueryEscrowAddressRequest{
 					PortId:    ibctesting.TransferPort,
-					ChannelId: ibctesting.FirstChannelID,
```

Comment:
This is still used in a lot of places. And I think in some of them still represents a valid identifier. I only changed tests that broke with the new functionality. I agree we should revisit it but it might be a longer work than it seems.

Reply:
I created #6765 , please feel free to edit to make it clearer, I wasn't sure how to phrase it. I haven't done any estimation yet on how much works is needed